lib/assets/stylesheets/overrides.scss

```diff
@@ -477,15 +482,24 @@ $tooltip-background: $grey;
 }
 
 /* Plan creation */
-.create-plan-or {
-  padding: 5px 0 0 15px;
-}
-.create-plan-checkbox {
-  margin-left: -40px;
-  margin-top: -5px;
-}
-.create-plan-mock {
-  margin-left: -40px;
+#new_plan {
+  .create-plan-or {
+    padding: 5px 0 0 15px;
+  }
+  .create-plan-checkbox {
+    margin-left: -40px;
+    margin-top: -5px;
+  }
+  .create-plan-mock {
+    margin-left: -40px;
+  }
+  .h2-label {
+    font-size: 30px;
+    font-weight: normal;
+  }
+  .red {
+    margin-left: 15px;
```

Comment:
it seems to me a very confusing class selector for the property styles that define. What about ml-15 instead?